### ISSUE TITLE
Some minor problems with sentences().toPastTense()

### ISSUE THREAD

**Issue by DeepakR-28** [spencermountain/compromise#802]
![image](https://user-images.githubusercontent.com/33338977/103405498-64426480-4b7d-11eb-9b7b-1734aeb9de83.png)

Was trying out the library by making this "Tense converter" thingy, as you can see the output says "**Will be you available**" instead of "**Will you be available**"

Is this where we "compromise" with other higher accuracy libraries? :p

```
here is the code snippet

window.futureTense = function(){
var text = nlp(document.getElementById("inputText").value).sentences().toFutureTense().text()
document.getElementById("outputText").innerHTML = text
}

```

Sorry if this is not the right place to ask this doubt, I was not sure where to ask.  

P.S thank you for making this cool library, Was very easy to work with it, and all the metadata is mind-blowing 🔥

**Comment by spencermountain** [comment]
hey Deepak, thanks, this certainly looks like a bug. Happy to fix it. Good find.

Yes, we're strong at conjugating single verbs, but admittedly there are weaknesses in flipping-around verb-phrases, especially in complex sentences. Whenever I start working on it, my brain begins to get melty. If you can find an alternative library, please share, I'd love to find one. This is something I'd like compromise to get good at this year, and help is welcomed. cheers

**Comment by DeepakR-28** [comment]
Exactly, Thanks for the reply :D . It converts the "Present Continuous" to "Past Tense" or "Future Tense" instead of the continuous forms. 
Will look it up somewhere else too. 
Thanks once again!
